src/xlsx/functions/conditional_formatting.ts

```diff
@@ -135,6 +139,25 @@ function cellRuleTypeAttributes(rule: CellIsRule): XMLAttributes {
   }
 }
 
+function addDataBarRule(cf: ConditionalFormat, rule: DataBarRule): XMLString {
+  const ruleAttributes = commonCfAttributes(cf);
+  ruleAttributes.push(["type", "dataBar"]);
+
+  // TODO ATM we do not support min and max values, so to have the same result
+  // in Excel, we export with min=0 and max=100
+  return escapeXml/*xml*/ `
```

Comment:
Data bars seem to be exported as gradient fill in excel rather than solid fill

Reply:
![image](https://github.com/user-attachments/assets/47b2fd82-c015-4d20-bd70-d82fe715b741)

Reply:
So it looks like Excel do not use the OpenXML databars, but its own custom databar type.
TBD if we want to stay standard, or if we want to match excel 
https://learn.microsoft.com/en-us/openspecs/office_standards/ms-xlsx/88c5ca0a-3a34-4fad-8c91-fdb4550d3fe1
![image](https://github.com/user-attachments/assets/a4cd5ca4-67a9-4a11-8e20-564a924fcc85)

Reply:
what version of excel where you using btw ? This is the result I get using Excel 365 desktop

Reply:
I used excel online.

> TBD if we want to stay standard, or if we want to match excel

Don't know, IMO we should stay standard but I don't have a strong opinion on this

Reply:
Sounds annoying without much gains TBH. I vote to ignore this. 

(I also have the `x14:dataBar` in excel online tho 😅 )

Reply:
So it's re-ready :D